src/pages/Introdece/Introduce.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import "./Introduce.scss";
+import { Link, useNavigate } from "react-router-dom";
+
+const Introduce = () => {
+  const handleInput = () => {
+    console.log("input 태그 동작");
+  };
+
+  // const handleButton = (event) => {
+  //   event.preventDefault();
+  //   console.log("button 태그 동작");
+  // };
+
+  const navigate = useNavigate();
+
+  return (
+    <div className="introduce">
+      <div className="introduceBox">
+        <img className="introduceImg" src="/images/jini.jpg" alt="jini" />
+        <span className="introduceName">최진이</span>
+        <span className="introduceNum">47기</span>
+        <form>
+          <input
+            className="introduceInput"
+            type="text"
+            placeholder="내용입력"
+            onChange={handleInput}
+          />
+          <button
+            className="introduceBtn"
+            type="sumbit"
+            onClick={() => {
+              navigate("./Product");
+            }}
```

Comment:
handleButton 함수내부에서 navigate를 활용할 수 있습니다!
또한 navigate의 인자는 이동해야 할 URL경로를 의미합니다. 여기서 말하는 경로는
해당 컴포넌트의 파일위치가 아닌 Router컴포넌트에서 지정한 path속성값과 같아야 합니다.
```suggestion
            onClick={() => {
              navigate("/product");
            }}
```
로 수정할 수 있겠네요!